src/tests/Microsoft.PowerFx.Core.Tests/IntellisenseTests/SignatureHelpTest.cs

```diff
@@ -13,7 +13,7 @@ namespace Microsoft.PowerFx.Tests.IntellisenseTests
 {
     public class SignatureHelpTest : IntellisenseTestBase
     {
-        private static readonly bool RegenerateSignatureHelp = false;
+        private const bool RegenerateSignatureHelp = false;
```

Comment:
>const [](http://example.com/codeflow?start=16&length=5)

`const` is stronger than `static readonly`
Const is just literals (like numbers). 
A static readonly could still point to something (a collection) that could get modified.

Reply:
This is in a test assembly, it shouldn't have been picked up by your new coverage (although, yes, good change still)